onward/app/views/fragments/richLinkBody.scala.html

```diff
@@ -43,23 +42,11 @@
 
         <div class="rich-link__header">
             <h1 class="rich-link__title">
-                @if(!experiments.ActiveExperiments.isParticipating(experiments.Garnett)) {
-                    @if(content.content.cardStyle == Comment ||
-                        (content.content.cardStyle == SpecialReport && content.content.hasTonalHeaderByline)
-                    ) {
-                        @fragments.inlineSvg("quote", "icon")
-                    }
-                } else {
-                    @if(content.content.cardStyle == Comment) {
-                        @fragments.inlineSvg("garnett-quote", "icon")
-                    }
+                @if(content.content.cardStyle == Comment) {
+                    @fragments.inlineSvg("garnett-quote", "icon")
```

Comment:
Perhaps we need a better name for this icon? The word `garnett` will probably be meaningless in 6 months.

Reply:
Is the old `quote.svg` used anywhere still? Could remove the old asset and rename `garnett-quote.svg` to `quote.svg` and?

Reply:
On the Opinion front?
![old quote](https://user-images.githubusercontent.com/6032869/35325985-3df49092-00ed-11e8-98af-17e471c66180.PNG)

Reply:
Yeah, it's still used in several places. I don't think it's ready to remove quite yet